doc/zstd_manual.html

```diff
@@ -76,7 +76,7 @@ <h1>zstd 1.4.3 Manual</h1>
 </b><p>  Compresses `src` content as a single zstd compressed frame into already allocated `dst`.
   Hint : compression runs faster if `dstCapacity` >=  `ZSTD_compressBound(srcSize)`.
   @return : compressed size written into `dst` (<= `dstCapacity),
-            or an error code if it fails (which can be tested using ZSTD_isError()). 
```

Comment:
Can you please revert your changes to this file. It looks like your editor probably stripped trailing whitespace.